serverless/event_sources/serverless-kafka-source.adoc

```diff
@@ -0,0 +1,19 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-kafka-source"]
+= Using a Kafka source
+include::modules/common-attributes.adoc[]
+:context: serverless-kafka-source
+toc::[]
+
+:FeatureName: Apache Kafka on {ServerlessProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+The Apache Kafka event source brings messages into Knative. It reads events from an Apache Kafka Cluster, and passes these events to an event sink, so that they can be consumed. You can use the `KafkaSource` event source with {ServerlessProductName}.
```

Comment:
Should "Cluster" be lower case in this context?

Reply:
To me the first of the two commas in the following sentence could be removed:

"It reads events from an Apache Kafka Cluster, and passes these events to an event sink, so that they can be consumed."